docs/terraformoptions.adoc

```diff
@@ -163,7 +163,12 @@ tags = {
 |false
 
 |`vcn_cidr`
-|The VCN's CIDR block. The CIDR block specified for the VCN must not overlap with the CIDR block of another network.
+|DEPRECATED: The VCN's CIDR block. Please use `vcn_cidrs` instead.
```

Comment:
As above. Let's make the breaking change considering we'll be going TF 1.0.0 and major version too